### ISSUE TITLE
make that table "delete" returns the count of deleted objects

### ISSUE THREAD

**Issue by jcouto** [datajoint/datajoint-python#1155]
### Problem
The table delete method returns the number of objects that fit the query as "delete_count". 
When the delete is aborted it returns a positive number.
 
### Workaround
If inside the inherited delete method, check the len(self) to know if deleted.

### Requirements
Return the number of deleted objects

### Justification
This can be used to perform actions only if the objects were deleted

### Related Errors
There are no errors but it is not intuitive that when no objects are deleted the function returns the number of objects that could have been deleted. :)

**Comment by dimitri-yatsenko** [comment]
I do not believe that the description is accurate. `delete_count` is returned by the database as the actual number of rows deleted when successful. If the delete is aborted, then no rows should be deleted. Deletes are atomic: all or none. I think the real issue is in the incorrect reporting of deleted records when the delete is rolled back. It should be set to zero.

**Comment by jcouto** [comment]
I agree, thanks.
